ddtrace/internal/packages.py

```diff
@@ -194,7 +194,7 @@ def _third_party_packages() -> set:
     ) - tp_config.excludes
 
 
-@cached()
+@cached(maxsize=65536)
```

Comment:
Because many file paths will map to the same package, we would expect this to be the larger cache needed. So as a rule of thumb, I'd say we can make the cache of functions that don't take a path as argument much smaller. And if we can also lower the size of path-based caches by a bit it would be great too, as paths could be quite long, and storing them when not really needed would cost some memory.

Reply:
Yes this is really a trade-off between CPU utilization and memory here. I can definitely lower the size of the various caches although the ones storing smaller objects use less memory so it might be okay for them to be bigger :)